File: tasty-fast/src/main/java/com/misha/tastyfast/requests/orderRequests/OrderDetailsResponse.java
package com.misha.tastyfast.requests.orderRequests;

import lombok.Builder;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;
@Data

public class OrderDetailsResponse {
    private Integer orderId;
    private LocalDateTime orderDate;
    private String status;
    private BigDecimal totalAmount;
    private Integer totalQuantity;
    private String orderType;
    private List<OrderItemResponse> items;

}

File: tasty-fast-ui/src/app/services/models/order-details-response.ts
/* tslint:disable */
/* eslint-disable */
import { OrderItemResponse } from '../models/order-item-response';
export interface OrderDetailsResponse {
  items?: Array<OrderItemResponse>;
  orderDate?: string;
  orderId?: number;
  orderType?: string;
  status?: string;
  totalAmount?: number;
  totalQuantity?: number;
}
